xbmc/music/tags/MusicInfoTag.cpp

```diff
@@ -381,10 +381,10 @@ void CMusicInfoTag::SetAlbumArtist(const std::string& strAlbumArtist)
   }
 }
 
-void CMusicInfoTag::SetAlbumArtist(const std::vector<std::string>& albumArtists)
+void CMusicInfoTag::SetAlbumArtist(const std::vector<std::string>& albumArtists, bool FillDesc /* = false*/)
 {
   m_albumArtist = albumArtists;
-  if (m_strAlbumArtistDesc.empty()) 
+  if (m_strAlbumArtistDesc.empty() || FillDesc) 
```

Comment:
Somehow it feels weird, that you say ```But the ALBUMARTISTS and ARTISTS tags mean that desc is not always a concatenation of the vector, so we have to set then separately.``` but we implicitly do it anyway if albumartistDesc is empty? I get that we only call them in that way from there. But is it really the best we can do?
Wouldn't it be more straight forward to just have fillDesc and ignore if it is empty?